toolchain/check/import_cpp.cpp

```diff
@@ -131,10 +139,146 @@ auto ImportCppFiles(Context& context, llvm::StringRef importing_file_path,
   name_scope.set_is_closed_import(true);
+  name_scope.set_is_cpp_scope(true);
+
+  *ast = std::move(generated_ast);
+  context.sem_ir().set_cpp_ast(ast->get());
 
   if (ast_has_error) {
     name_scope.set_has_error();
   }
 }
 
+static auto ClangLookup(Context& context, SemIR::NameId name_id)
+    -> std::optional<clang::LookupResult> {
+  clang::ASTUnit* ast = context.sem_ir().cpp_ast();
+  CARBON_CHECK(ast);
+  CARBON_CHECK(ast->hasSema());
+  clang::Sema& sema = ast->getSema();
+
+  auto name = context.names().GetAsStringIfIdentifier(name_id);
+  CARBON_CHECK(name);
+
+  clang::LookupResult lookup(
+      sema,
+      clang::DeclarationNameInfo(
+          clang::DeclarationName(
+              sema.getPreprocessor().getIdentifierInfo(*name)),
+          clang::SourceLocation()),
+      clang::Sema::LookupNameKind::LookupOrdinaryName);
+
+  if (!sema.LookupQualifiedName(
+          lookup, ast->getASTContext().getTranslationUnitDecl())) {
+    return std::nullopt;
+  }
+
+  return lookup;
+}
+
+static auto ImportFunctionDecl(Context& context, SemIR::LocId loc_id,
+                               SemIR::NameScopeId scope_id,
+                               SemIR::NameId name_id,
+                               const clang::FunctionDecl* clang_decl)
+    -> SemIR::InstId {
+  if (clang_decl->isVariadic()) {
+    context.TODO(loc_id, "Unsupported: Variadic function");
+    return SemIR::InstId::None;
+  }
+  if (!clang_decl->isGlobal()) {
+    context.TODO(loc_id, "Unsupported: Non-global function");
+    return SemIR::InstId::None;
+  }
+  if (clang_decl->getTemplatedKind() != clang::FunctionDecl::TK_NonTemplate) {
+    context.TODO(loc_id, "Unsupported: Template function");
+    return SemIR::InstId::None;
+  }
+  if (!clang_decl->param_empty()) {
+    context.TODO(loc_id, "Unsupported: Function with parameters");
+    return SemIR::InstId::None;
+  }
+  if (!clang_decl->getReturnType()->isVoidType()) {
+    context.TODO(loc_id, "Unsupported: Function with non-void return type");
+    return SemIR::InstId::None;
+  }
+
+  auto function_decl = SemIR::FunctionDecl{
+      SemIR::TypeId::None, SemIR::FunctionId::None, SemIR::InstBlockId::Empty};
+  auto decl_id = AddPlaceholderInst(
+      context, SemIR::LocIdAndInst(Parse::NodeId::None, function_decl));
+
+  auto function_info = SemIR::Function{
+      SemIR::EntityWithParamsBase{
+          .name_id = name_id,
+          .parent_scope_id = scope_id,
+          .generic_id = SemIR::GenericId::None,
+          .first_param_node_id = Parse::NodeId::None,
+          .last_param_node_id = Parse::NodeId::None,
+          .pattern_block_id = SemIR::InstBlockId::Empty,
+          .implicit_param_patterns_id = SemIR::InstBlockId::Empty,
+          .param_patterns_id = SemIR::InstBlockId::Empty,
+          .call_params_id = SemIR::InstBlockId::Empty,
+          .is_extern = false,
+          .extern_library_id = SemIR::LibraryNameId ::None,
```

Comment:
```suggestion
          .extern_library_id = SemIR::LibraryNameId::None,
```

I'm surprised clang-format didn't fix this...?

Reply:
Done.
Me too.
I've noticed such issues when auto-formatting the code, but I guess I missed this one.